code/game/machinery/suit_storage_unit.dm

```diff
@@ -609,7 +609,7 @@
 	//Departments that the cycler can paint suits to look like.
 	var/list/departments = list("Engineering","Mining","Medical","Security","Atmos")
 	//Species that the suits can be configured to fit.
-	var/list/species = list("Human","Skrell","Unathi","Tajara", "Vaurca", "Machine")
+	var/list/species = list(SPECIES_HUMAN,SPECIES_SKRELL,SPECIES_UNATHI,SPECIES_TAJARA, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BREEDER, SPECIES_VAURCA_WARFORM, SPECIES_IPC)
```

Comment:
This uses bodytypes, not species names. Also, vaurca warform and breeders are not playable species and should not show up here.

Reply:
>_< Confusing when things that say 'species' mean 'bodytype'. Thanks.